src/field/pokemon.ts

```diff
@@ -217,6 +217,28 @@ interface AbilityData {
   passive: boolean;
 }
 
+interface EffectiveStatOptions {
+  /** The target {@linkcode Pokemon} */
```

Comment:
```suggestion
  /**
   * The opposing {@linkcode Pokemon}, usually
   * involved in an incoming or outgoing attack
   */
```
The current doc isn't accurate because, depending on the Stat being calculated, `opponent` can be the attacker or defender.